src/socket.ts

```diff
@@ -698,18 +737,9 @@ export class OutOfWorkerRelay extends EventDispatcher {
         isAutoFlushIncoming: boolean, isAutoFlushOutgoing: boolean ) {
         super();
         const self = this;
-        self.strRelayName = ( strRelayName != null && strRelayName != undefined &&
-            typeof strRelayName === "string" && strRelayName.length > 0 )
-            ? ( "" + strRelayName )
-            : "unnamed";
-        self.isAutoFlushIncoming =
-            ( isAutoFlushIncoming == null || isAutoFlushIncoming == undefined )
-                ? true
-                : ( !!isAutoFlushIncoming );
-        self.isAutoFlushOutgoing =
-            ( isAutoFlushOutgoing == null || isAutoFlushOutgoing == undefined )
-                ? true
-                : ( !!( isAutoFlushOutgoing ) );
+        self.strRelayName = strRelayName ? strRelayName.toString() : "unnamed";
+        self.isAutoFlushIncoming = isAutoFlushIncoming ? true : ( !!isAutoFlushIncoming );
+        self.isAutoFlushOutgoing = isAutoFlushOutgoing ? true : ( !!( isAutoFlushOutgoing ) );
```

Comment:
Why do you need extra brackets in `( !!( isAutoFlushOutgoing ) )` ?

Reply:
Fixed misprint